news/_posts/2025-03-03-release-202500303.md

```diff
@@ -0,0 +1,18 @@
+---
+kind: article
+title: "Release 202500303"
+author: "PunkPun"
+created_at: 2025-03-03 16:00 +0000
+disqus_id: "release-202500303"
+permalink: "/news/release-202500303/"
+---
+
+Today is the day that many have been waiting for: the new stable OpenRA Release 20250303 is now available!
+
+This brings the public release up to date with development changes from August 2023 through to January 2025. Three months of public playtesting have allowed us to polish these changes into the release today.
+
+This release brings a newly revamped Map Editor, a switch to high-quality Dune 2000 assets, a TiberiAm Dawn encyclopedia, new missions and all around usability and performance improvements. For details see the previous playtest news posts [[one]({{ '/news/playtest-20241116/' | relative_url }}), [two]({{ '/news/playtest-20241228/' | relative_url }}), [three]({{ '/news/playtest-20250220/' | relative_url }})] and, as always, for a full overview - [the full changelog](https://github.com/OpenRA/OpenRA/wiki/Changelog/664a69f1b220683a5174017b2b694201ed6c4ba4).
```

Comment:
Repeating "brings" again here feels off. How about "This release revamps the map editor", switches to ...?